Misc/NEWS.d/next/Library/2024-11-29-11-04-15.gh-issue-127383.uH22xx.rst

```diff
@@ -0,0 +1 @@
+Add the latest ``RDS`` macro constants to the :mod:`socket`.
```

Comment:
This NEWS entry does not add any value. Either a) word the NEWS entry so it spells out the change, or b) remove the NEWS entry. As previously mentioned multiple times, please see the devguide on _how_ and _when_ to write NEWS entries.